README.md

```diff
@@ -1,6 +1,6 @@
 # generator-standard-readme
 
-[![Build Status](https://travis-ci.org/RichardLitt/generator-standard-readme.svg?branch=master)](https://travis-ci.org/RichardLitt/generator-standard-readme)
+[![Build status](https://github.com/RichardLitt/generator-standard-readme/actions/workflows/ci.yml/badge.svg)](https://github.com/RichardLitt/generator-standard-readme/actions/workflows/ci.yml)
```

Comment:
I figured that using the GitHub Actions workflow would suffice here. I wasn't around as a developer when Travis CI was more commonly used, so I'm unsure about how attached we are to it.

Reply:
Not attached. This works.